Library/Formula/unrar.rb

```diff
@@ -15,6 +15,11 @@ class Unrar < Formula
   def install
     system "make"
     bin.install 'unrar'
+    system "make", "clean"
+
+    system "make", "lib"
+    mv "./libunrar.so", "./libunrar.dylib"
```

Comment:
Just drop them a polite email upstream asking if it'd be possible to compile ` dylibs ` on OS X rather than ` .so ` files, as the former is OS X's standard preferred library, and then link to that email in the formula if possible (i.e. if the email is sent to a mailing list) :smiley_cat: